### ISSUE TITLE
Review how accessible editor place holders are

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#147107]
@isidorn need your brain here

When I enabled editor place holders to appear when an editor fails to open, I also pushed a change to remove the error toast that would appear in that case, so as a user all you get is an editor that you first have to figure out that an error is presented.

For someone that relies on screen reader this maybe now harder to figure out compared to before, because every notification toast is announced as alert and thus was clearly different from a normal editor opening successfully.

I think the question is mainly: what should the accessible text be announcing in case of an error and should we also send out an ARIA alert like we did before with the toasts?

Here is a recording how macOS behaves:

![Kapture 2022-04-08 at 20 03 48](https://user-images.githubusercontent.com/900690/162497211-4c777f44-c777-4334-a96d-a6a4fad3c3db.gif)

Btw we have the exact same problem already for workspace trust editors where we were never showing a toast, so I wonder if this was ever discussed @sbatten @lszomoru ?

![image](https://user-images.githubusercontent.com/900690/162497280-4c7b36a7-5572-4d8b-827a-63fff60189d9.png)

**Comment by bpasero** [comment]
Oh! I just figured out I can tweak the label via ARIA, can do that for all our placeholders:

![image](https://user-images.githubusercontent.com/900690/162497764-e72f1249-a1a5-4183-81f3-15d93cb74c91.png)

I think we are good then?

**Comment by isidorn** [comment]
@bpasero love your fix. It is always better to not use aria alerts, but labels based on focus.
So yeas we are good. Ideally `tab` goes to the actions, and if that is the case I think it is fine that we do not announce in advance number of actions.
